tests/python/unittest/test_meta_schedule_database.py

```diff
@@ -68,9 +76,78 @@ def main(a: T.handle, b: T.handle, d: T.handle) -> None:  # pylint: disable=no-s
                 D[vi, vj] = T.max(C[vi, vj], 0.0)
 
 
-# fmt: on
-# pylint: enable=invalid-name,no-member,line-too-long,too-many-nested-blocks,no-self-argument
-
+@tvm.script.ir_module
+class MobileNetConv2d:
```

Comment:
first of all, this PR fixes a TIR issue, so it has nothing to do with MetaSchedule fundamentally; second, we should focus on "minimal" reproducible example while it's indeed not minimal enough.

alternatively, I would suggest
1) creating a unittest in `test_tir_schedule_split_fuse.py`
2) the IRModule contains a single loop
3) the schedule contains a single `split`, whose factor is `factors=[None, IntImm("int64", some-number)]`
4) check if the loop extent remains int32